object detection/custom detector/python/pytorch_yolov7/README.md

```diff
@@ -9,28 +9,28 @@ This sample shows how to detect custom objects using the official Pytorch implem
 
 ## Setting up
 
- - Clone Yolov5 into the current folder
+ - Clone Yolov7 into the current folder
 
 ```sh
-git clone https://github.com/ultralytics/yolov5
+git clone https://github.com/WongKinYiu/yolov7
 # Install the dependencies if needed
-cd yolov5
+cd yolov7
 pip install -r requirements.txt
 ```
 
-- Download a model file (or prepare your own) https://github.com/ultralytics/yolov5/releases
+- Download a model file (or prepare your own) https://github.com/WongKinYiu/yolov7/releases/tag/v0.1
 
 ```
 # Downloading by commmand line
-wget https://github.com/ultralytics/yolov5/releases/download/v6.0/yolov5m.pt
+wget https://github.com/WongKinYiu/yolov7/releases/download/v0.1/yolov7.pt
 ```
 
 ## Run the program
 
 *NOTE: The ZED v1 is not compatible with this module*
 
 ```
-python detector.py --weights yolov5m.pt # [--img_size 512 --conf_thres 0.1 --svo path/to/file.svo]
+python detector.py --weights yolov7.pt # [--img_size 512 --conf_thres 0.1 --svo path/to/file.svo]
```

Comment:
Maybe set --img_size to 640 as it is model input size or yolov7?

Reply:
Your issue seems to be related to hw constraints. We used a smaller model and we didn't face any problem in a laptop with 6gb vram. Regarding your comment, it makes sense but these are the official instructions.